docs/source/quickstart.rst

```diff
@@ -1,81 +1,187 @@
-    -  A script to run the pipeline, ``run_eureka.py``. 
+    -  ``*.ecf``: These are ``Eureka!`` control files, and contain input parameters required to run each stage of the pipeline. For more detail on the ecf parameters for each stage, see :ref:`here<ecf>`.
+    -  ``*.epf``: This is a ``Eureka!`` parameter file, and describes the initial guesses and priors to be used when performing light curve fitting (Stage 5).
+    -  ``run_eureka.py``: A script to run the ``Eureka!`` pipeline. 
 
+3.2 Customise the demo files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-3.2  To set up a run, first decide which stage you want to start with. For example, if you download Stage 2 data, you will start with ``Eureka!`` Stage 3. 
+You might notice that not all of the demo files will be applicable to every dataset, either because they are tailored to a specific instrument, or because they are for a ``Eureka!`` pipeline stage that precedes the input data. This is the case for the NIRSpec data being used here, which as a ``*rateints.fits`` file (more information on JWST pipeline products `here <https://jwst-pipeline.readthedocs.io/en/latest/jwst/data_products/product_types.html>`_) has already been processed through an equivalent to Stage 1 of ``Eureka!``.
 
-Next, add an event label to the ecf filenames. If you're working with data for WASP-39b, the event label could be ``wasp39b``. Modify the file names of the templates that are appropriate for your stage and instrument mode. For example, if you're starting with Stage 2 NIRSPec data, you would update the following filenames: 
+So, let's only copy over the specific files needed to process this NIRSpec dataset further. Given that the dataset contains a transit for WASP-39b, let's also change some of the default filenames to something a little more informative:
 
 .. code-block::
 
-	mv S3_nirspec_fs_template.ecf S3_wasp39b.ecf
-	mv S4_template.ecf S4_wasp39b.ecf
-	mv S5_template.ecf S5_wasp39b.ecf
-	mv S6_template.ecf S6_wasp39b.ecf
+	cp demos/run_eureka.py .
+	cp demos/S2_nirspec_fs_template.ecf S2_wasp39b.ecf
+	cp demos/S3_nirspec_fs_template.ecf S3_wasp39b.ecf
+	cp demos/S4_template.ecf S4_wasp39b.ecf
+	cp demos/S5_template.ecf S5_wasp39b.ecf
+	cp demos/S5_fit_par_template.epf S5_fit_par_wasp39b.ecf
+	cp demos/S6_template.ecf S6_wasp39b.ecf
 
-Next, update the ``topdir``, ``inputdir``, and ``outputdir`` in each ``.ecf`` file. For each stage, the ``inputdir`` is the previous stage, e.g.:
+Notice that all of the ``*.ecf`` files have a common ``wasp39b`` string. It's useful to keep this homogenous across files as it is what ``Eureka!`` interprets as an "event label", and is used to locate specific input files when running the pipeline. To see this more clearly, open up the ``run_eureka.py`` file and look at how the individual stages are being called. While you're here, modify the ``eventlabel`` string directly to match the chosen naming:
```

Comment:
Here I'd tell people to comment out the Stage 1 line since they're starting from a Stage 1 output